.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - 9
```

Comment:
looks like some modules are only compatible with LTS versions of node so I also removed 9 & 7
> error get-caller-file@2.0.5: The engine "node" is incompatible with this module. Expected version "6.* || 8.* || >= 10.*".